ui/app.js

```diff
@@ -5736,3 +5774,75 @@ $(".map-clear").on('click', function() {
       ClearMap();
     }, 1500);
 });
+
+function searchOfficerResults(result) {
+  canSearchForOfficer = true;
+  $(".leaderboard-items").empty();
+
+  if (result.length < 1) {
+    $(".leaderboard-items").html(
+      `
+                      <div class="leaderboard-item" data-id="0">
+
+                          <div style="display: flex; flex-direction: column; margin-top: 2.5px; margin-left: 5px; width: 100%; padding: 5px;">
+                          <div style="display: flex; flex-direction: column;">
+                              <div class="leaderboard-item-title">No Officer Matching that search</div>
+                              </div>
+                              <div class="leaderboard-bottom-info">
+                              </div>
+                          </div>
+                      </div>
+              `
+    );
+    return true;
+  }
+
+
+  let leaderboardOfficerHTML = "";
+
+  result.forEach((value) => {
+    let charinfo = value.charinfo;
+    let metadata = value.metadata;
+
+    if (typeof value.charinfo == "string") {
+      charinfo = JSON.parse(charinfo);
+    }
+  
+    if (typeof value.metadata == "string") {
+      metadata = JSON.parse(metadata);
+    }
+
+    if (!metadata) {
+      metadata = {};
+    }
+
+    if (!metadata.licences) {
+      metadata.licences = {};
+    }
+  
+    let name = charinfo.firstname + " " + charinfo.lastname;

```

Comment:
A better way of doing this to make it more readable would be:

```suggestion
    let name = `${charinfo.firstname} ${charinfo.lastname}`;
```

Reply:
done <3